driver-core/src/main/com/mongodb/internal/connection/AsynchronousChannelStream.java

```diff
@@ -88,7 +89,7 @@ protected void setChannel(final ExtendedAsynchronousByteChannel channel) {
 
     @Override
     public void writeAsync(final List<ByteBuf> buffers, final OperationContext operationContext,
-            final AsyncCompletionHandler<Void> handler) {
+                           final AsyncCompletionHandler<Void> handler) {
```

Comment:
Do we really need the above change? Seems the aim is to vertical alignment? Usually parameters are put into a new  line because the previous parameters are too long, and maintaining vertical alignment won't help when the remaining parameters are long and need to be split again.